### ISSUE TITLE
Button to save basic auth creds is inaccessible to NVDA on Windows 10

### ISSUE THREAD

**Issue by tombye** [alphagov/reported-bugs#62]
**Upstream bug:** https://bugzilla.mozilla.org/show_bug.cgi?id=1729670
**Date:** 2021-09-08
**Reported by:** Tom Byers
**Related to:** Not applicable

---

### Overview

The NVDA screen reader cannot access the 'Save' button on the basic auth dialog for saving credentials.

Note: I was using NVDA 2021.1 through assistivelabs.com.

### Expected result

You should be able to save the credentials you entered by using the 'Save' button.

### Actual result

You can access all the other fields but not the 'Save' button.

**Comment by tombye** [comment]
I am closing this issue because, rather embarrassingly, I realised that you can access the 'Save' button, it's just that it is after the 'Don't save' button in the tab order. I just didn't tab far enough to notice 😬.

I've updated the bug with comments to that effect.
